Sources/Sentry/SentrySessionReplayIntegration.m

```diff
@@ -615,6 +617,33 @@ - (nullable NSString *)currentScreenNameForSessionReplay
                .firstObject;
 }
 
+- (void)showMaskPreview:(CGFloat)opacity
```

Comment:
`m`: It bugs me a bit, that we don't have tests for that code. What can we do to increase the test coverage?

Reply:
We do have tests for showMaskPreview. With and without the debugger attached.